### ISSUE TITLE
 iOS: Against the YouTube Terms of Service

### ISSUE THREAD

**Issue by hoanglm4** [hoanglm4/flutter_youtube_view#8]
In iOS: We using [XCDYouTubeKit](https://github.com/0xced/XCDYouTubeKit) is against the YouTube Terms of Service. So we will change to use https://github.com/rinov/YoutubeKit

Supported: https://github.com/rinov/YoutubeKit/blob/master/README.md

**Comment by docaohuynh** [comment]
Hi @hoanglm4 
Is posible to play audio in background?

**Comment by hoanglm4** [comment]
Hello @docaohuynh 

Currently, this lib can not play video in background

**Comment by docaohuynh** [comment]
HI @hoanglm4 
I have an error when build on IOS. How do I solve it
```
Xcode build done.                                            7.5s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    In file included from /MyData/myProject/OwnSource/Mobile/MyProject/ieltseccyl/ios/Pods/XCDYouTubeKit/XCDYouTubeKit/XCDYouTubeVideo.m:5:
    /MyData/myProject/OwnSource/Mobile/MyProject/ieltseccyl/ios/Pods/XCDYouTubeKit/XCDYouTubeKit/XCDYouTubeVideo+Private.h:5:9: fatal error: 'XCDYouTubeKit/XCDYouTubeVideo.h' file not found
    #import <XCDYouTubeKit/XCDYouTubeVideo.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.

Encountered error while building for device.
```

**Comment by docaohuynh** [comment]
I have built with Xcode it work now :)

**Comment by hoanglm4** [comment]
Available on: `flutter_youtube_view: ^1.1.4`
